frontends/PyCDE/src/pycde/module.py

```diff
@@ -345,32 +345,26 @@ def __call__(self, op):
     else:
       module_name = self.create_module_name(mod, op)
 
-    if module_name not in self.generated_modules:
+    if module_name not in _Generate.generated_modules:
       with mlir.ir.InsertionPoint(mod.regions[0].blocks[0]):
         gen_mod = ModuleDefinition(self.modcls,
                                    module_name,
                                    input_ports=input_ports,
                                    output_ports=output_ports,
                                    body_builder=self.gen_func)
-        self.generated_modules[module_name] = gen_mod
+        _Generate.generated_modules[module_name] = gen_mod
 
     # Build a replacement instance at the op to be replaced.
     with mlir.ir.InsertionPoint(op):
       mapping = {name.value: op.operands[i] for i, name in enumerate(op_names)}
-      inst = self.generated_modules[module_name].create(op.name,
-                                                        **mapping).operation
+      inst = _Generate.generated_modules[module_name].create(
+          op.name, **mapping).operation
       for (name, attr) in attrs.items():
         inst.attributes[name] = attr
       return inst
 
   def create_module_name(self, mod, op):
-    name = op.name.replace("pycde.", "")
-    existing_module_names = set(
```

Comment:
I added this cause I thought it was going to help the common case... if it's not the common case let's remove it.

Reply:
Yeah, the problem is that with all the modules we're creating they're likely to alias with hand-written RTL modules. Especially when a user is porting a design over and being faithful to the RTL.